Otel quarkus vertx

jasondlee opened the pull request:
Thanks for submitting your Pull Request!

Please delete this text, and add a link to the Jira issue solved by this PR.

If this PR is not for the 'main' branch you must add a link to the equivalent change in 'main'.

Remember to use the Jira issue ID in the PR title and any commits.

bstansberry commented on galleon-pack/galleon-shared/pom.xml:
```diff
@@ -49,6 +49,17 @@
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
```
Please remove the exclusions. The bom handles that in dependencyManagement.

bstansberry commented on galleon-pack/galleon-shared/pom.xml:
```diff
@@ -447,6 +460,26 @@
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
```
Same comment as above.

bstansberry commented on galleon-pack/galleon-shared/pom.xml:
```diff
@@ -447,6 +460,26 @@
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
```
Same comment as above.

bstansberry commented on galleon-pack/galleon-shared/src/main/resources/license/licenses.xml:
```diff
@@ -558,16 +558,6 @@
-                    <url>http://www.apache.org/licenses/LICENSE-2.0</url>
-                </license>
-            </licenses>
-        </dependency>
```
I expect there may need to be more stuff done to this file, although maybe not as how this works is being redone.

jasondlee committed:
[otel_quarkus_vertx] Remove okhttp/kotlin dependency

Update SR Otel to 2.6.0
Modify opentelemetry subsystem to use new SR-provided exporters

jasondlee committed:
[WFLY-18448] Update SmallRye OpenTelemetry to 2.5.1

Bump SR Otel to 2.5.1
Update modules to reflect upstream changes
Exclude challenged TCK test

jasondlee committed:
[otel_quarkus_vertx]

jasondlee committed:
[otel_quarkus_vertx]

Dependency and module clean up
